RxSwift/Disposables/DisposeBag.swift

```diff
@@ -82,3 +82,31 @@ public final class DisposeBag: DisposeBase {
         dispose()
     }
 }
+
+extension DisposeBag {
+    /// Convenience function allows a list of disposables to be gathered for disposal.
+    ///
+    /// With disposeBag.bag:
```

Comment:
I would remove this explanation and just leave
```
Convenience function allows a list of disposables to be gathered for disposal.

/// - parameter disposables: List of disposables to add.
```

This is a pretty simple API.